### PR TITLE
NAS-125571 / 24.04 / Storj Backup Widget

### PR DESCRIPTION
**Pull request by bvasilenko**
**Summary**

4 states of the widget according to specs in the ticket:

image

image

image

image

**Testing**

1. Test the widget with different types of data protection jobs ensure that widget displays correct set of tiles, and  the summarized stats are also correct.

2. Test the hyperlinks to create new jobs.

### CODE REVIEW

File: src/app/pages/dashboard/components/widget-backup/widget-backup.component.html [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+                name="mdi-check-circle"
+                class="icon healthy"
+              ></ix-icon>
+              <ix-icon
+                *ngIf="!!failedCount"
+                name="mdi-alert"
+                class="icon warning"
+              ></ix-icon>
+              <span *ngIf="!!failedCount">
+                {{ '{failed} of {all} tasks failed' | translate : { failed: failedCount, all: allCount } }}
```
Comment (denysbutenko) [inline src/app/pages/dashboard/components/widget-backup/widget-backup.component.html:47]: I think variable names are better self-explanatory than placeholders.
Let's update placeholder names to match variables.

Also, `{all} tasks` need plural support.

**Review by denysbutenko — CHANGES_REQUESTED** [review]
image

It looks good but requires some improvements.
The backup widget is missing on mobile screens.

File: src/app/pages/dashboard/components/widget-backup/widget-backup.component.html (hunk 1 above)
Comment (denysbutenko) [reply src/app/pages/dashboard/components/widget-backup/widget-backup.component.html:47]: Plural support seems required for each occurrence of `tasks` in this PR. Please take a look.

**Comment by bvasilenko** [comment]
@denysbutenko the requested changes should be addressed now

**Review by denysbutenko — APPROVED** [review]
Looks good, except few things:

The widget is still missing on mobile screens.

It's unclear why the text is being truncated. If there is something that should appear on the opposite side – let's add a title for links, at least. 
image

**Comment by bvasilenko** [comment]
@denysbutenko now widget is optimized for mobile:

image

other requested changes should be addressed as well

**Comment by denysbutenko** [comment]
> @denysbutenko now widget is optimized for mobile:
> 
> image
> 
> other requested changes should be addressed as well

PR is approved. Thanks!

**Pull request merged**